jose.py

```diff
@@ -139,14 +137,15 @@ def encrypt(claims, jwk, adata='', add_header=None, alg='RSA-OAEP',
     assert _TEMP_VER_KEY not in claims
     claims[_TEMP_VER_KEY] = _TEMP_VER
 
-    header = dict((add_header or {}).items() + [
-        ('enc', enc), ('alg', alg)])
+    header = dict(
+        list((add_header or {}).items()) + [('enc', enc), ('alg', alg)]
+    )
```

Comment:
You didn't add this but it looks a bit ugly.
It would look much better this way: `header = dict(add_header or {}, enc=enc, alg=alg)`